embassy-nrf/src/lib.rs

```diff
@@ -115,6 +115,9 @@ pub use chip::pac;
 #[cfg(not(feature = "unstable-pac"))]
 pub(crate) use chip::pac;
 
+#[cfg(feature = "unstable-pac")]
```

Comment:
I don't think gating it behind `unstable-` is needed, it's part of the stable public API so it's not like we're going to change it.

Reply:
I made it so because it cannot be used without accessing PAC. But you're right that it is stable and public.